CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.6.9
+
+* Require `2.0.0` or newer SDK.
+* Drop unnecessary `new` and `const`.
```

Comment:
This line can be dropped, right? Doesn't affect end users.

Reply:
🤷‍♂️ Jake asked for an entry

https://github.com/dart-lang/stream_channel/pull/37#discussion_r220599243

Reply:
@jakemac53 – you silly